src/plugins/ui-router-translations/ui-router-translations.js

```diff
@@ -0,0 +1,70 @@
+const { createFilter } = require('rollup-pluginutils');
+const _ = require('lodash');
+const fs = require('fs');
+const glob = require('glob');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const translationTools = require('@ovh-ux/translations-build-tools');
+const utils = require('./utils');
+const config = require('../../config');
+
+const stateHandler = (source, options) => {
+  let result = source;
```

Comment:
and source become result, and result become source and ... :D

Please refer to my previous comment :)